### ISSUE TITLE
Error after unsuccessful project download

### ISSUE THREAD

**Issue by VitorVieiraZ** [MerginMaps/qgis-plugin#649]
If we start downloading a project and close QGIS during the process, reopening QGIS and attempting to download the incomplete project again results in an error stating that the project cannot be downloaded because a folder for it already exists.

**Comment by ValentinBuira** [comment]
Yes we don't handle partial download right now, the simplest workaround right now, is to clear the folder and start again the download from scratch
